src/components/WalletDropdown/MiniPortfolio/NFT.tsx

```diff
@@ -62,10 +55,9 @@ export function NFT({
         display={{ disabledInfo: true }}
         isSelected={false}
         isDisabled={false}
-        selectAsset={navigateToNFTDetails}
-        unselectAsset={() => {
-          /* */
-        }}
+        selectAsset={() => null}
+        unselectAsset={() => null}
```

Comment:
it seems like we were already passing a function that didn't do anything here, but i think that if we're okay with inaction on a select/unselect, these props shouldn't be a required function in `NftCard` and instead should be optional props (with a comensurate `if (selectAsset) selectAsset()` in the component click handler. what do you think?

Reply:
i agree. the NFT card seems like one of those components that has grown in complexity so i subconsciously erred on the side of "don't mess with it". but i think making these props optional is pretty safe.

Reply:
that makes sense - i'd like to chip away at that complexity bit by bit too so it's less scary to change